### PR TITLE
MAD-8849: Build Portal API Documentation: Package Module

### PR DESCRIPTION
**Pull request by chittaranjan-behera**
MAD-8849: Build Portal API Documentation: Package Module

### CODE REVIEW

File: package/addon/list.md [hunk 1]
```diff
@@ -9,15 +9,13 @@ package-addon:list
 Lists package addons (typically, belonging to a particular package). 
 Only addons that can be purchased by the client making the request will be returned.
 
-**Endpoint**:  GET /v1/package/addon
+**Endpoint**:  GET /v1/package/addon/{id}
 
-**Alternate Endpoint**:  GET /v1/package/{id}/addon[/{type}]
-
-This alternate endpoint is sugar for `GET /v1/package/addon?filter[package_id]={id}&filter[type]={type}` and will serve identical responses. The `type` parameter is optional.
+This alternate endpoint is sugar for `GET /v1/package/addon/{id}?filter[package_id]={id}&filter[type]={type}` and will serve identical responses. The `type` parameter is optional.
```
Comment (adrian-enspired) [inline package/addon/list.md:14]: revert; the original description was correct

File: package/addon/list.md [hunk 2]
```diff
@@ -9,15 +9,13 @@ package-addon:list
...
 
 **Alternate Endpoint**:  GET /v1/package/addon/{type}
 
-This alternate endpoint is sugar for `GET /v1/package/addon?filter[type]={type}` and will serve identical responses.
+This alternate endpoint is sugar for `GET /v1/package/addon/{type}?filter[type]={type}` and will serve identical responses.
```
Comment (adrian-enspired) [inline package/addon/list.md:18]: revert; the original description was correct

**Review by adrian-enspired — CHANGES_REQUESTED** [review]

File: package/addon/list.md (hunk 1 above)
Comment (chittaranjan-behera) [reply package/addon/list.md]: reverted

File: package/addon/list.md (hunk 2 above)
Comment (chittaranjan-behera) [reply package/addon/list.md]: reverted to the original description

**Pull request merged**